projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/boost-score-tooltip.tsx

```diff
@@ -0,0 +1,48 @@
+import { Popover } from '@wordpress/components';
+import React from 'react';
+import type { FC, ReactNode } from 'react';
+
+type VirtualElement = Pick< Element, 'getBoundingClientRect' > & {
+	ownerDocument?: Document;
+};
+
+interface Props {
+	isVisible: boolean;
+	children: ReactNode;
+	placement?:
+		| 'top'
+		| 'top-start'
+		| 'top-end'
+		| 'right'
+		| 'right-start'
+		| 'right-end'
+		| 'bottom'
+		| 'bottom-start'
+		| 'bottom-end'
+		| 'left'
+		| 'left-start'
+		| 'left-end'
+		| 'overlay';
+	offset?: number;
+	anchor?: Element | VirtualElement;
+}
+
+const BoostScoreTooltip: FC< Props > = ( {
+	isVisible,
+	children,
+	placement = 'right',
+	offset,
+	anchor,
+} ) => {
+	return (
+		isVisible && (
+			<div className={ 'boost-score-tooltip' }>
+				<Popover placement={ placement } noArrow={ false } offset={ offset } anchor={ anchor }>
+					{ children }
+				</Popover>
+			</div>
+		)
+	);
+};
+
+export default BoostScoreTooltip;
```

Comment:
I question whether this boost specific Popover tooltip component needs to exist 🤔 

It seems as though all props except `isVisible` are passed straight to the Popover component, and the only boost specific thing in here is the `boost-score-tooltip` class on the wrapping div. I am looking at the [Popover component here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/popover/index.jsx#L25) and I see that it accepts a `className` prop that we could pass along instead of wrapping in a `<div>`. It looks like it also accepts an `isVisilbe` prop [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/popover/index.jsx#L411) that we could pass directly to the component. What do you think? I may be missing something here

Reply:
Yes, this is a good point. We can use the Popover component directly, no need to wrap it within a separate component. 👍